llvm/lib/DWARFLinker/Utils.cpp

```diff
@@ -5,3 +5,5 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
+
+#include "llvm/DWARFLinker/Utils.h"
```

Comment:
Do you intend to add any code to this file? If it's empty, we can delete it

Reply:
yes, I am going to add code to this file in the future. Another thing is that currently this is the only .cpp file in the library. Without any .cpp file it would be necessary to write more complex CMakeLists.txt. It is just easier to have just empty .cpp.